Neos.Ui/core/src/application/LinkTypes/Node/Node.tsx

```diff
@@ -55,26 +70,30 @@ export const Node = makeLinkType<NodeLinkModel, NodeLinkOptions>('Sitegeist.Arch
             const cacheIdentifier = `${identifier}@${siteNodeContextPath.context}`;
 
             if (nodeCache.has(cacheIdentifier)) {
```

Comment:
when you review this make sure that after deleting a node, you need to reload the ui (that way the nodeCache is invalidated)

otherwise it would be a super rare real work use-case:
> 1. link on siteA to a siteB
> 2. delete the siteB
> 3. got back to siteA to modify the link to the deleted siteB


the code that would be required for proper cache invalidation for the nodeCache is disproportionate compared to the benefit.

so i decided to not fix this bug for now.

(so please releload after step 2 ^^)